### ISSUE TITLE
OSProcess.getThreadCount() and others return zero

### ISSUE THREAD

**Issue by LGH** [oshi/oshi#610]
I revisited an old side project and decided to update from oshi-3.4.4 to oshi-3.8.1, and now the OSProcess methods return static values (mostly zero).

This is the calling code:
```java
logger.trace("oshi pid={}, threads={}, getKernelTime()={}, getUserTime()={}",
                                osProcess.getProcessID(),
                                osProcess.getThreadCount(),
                                osProcess.getKernelTime(),
                                osProcess.getUserTime()
                            );
```

With oshi-3.4.4:
```
2018-09-06 11:59:47.484 TRACE 10986 --- [pool-1-thread-1] c.k.m.myapp.tasks.ProcessMonitorTask    : oshi pid=18921, threads=118, getKernelTime()=369266, getUserTime()=886783966
2018-09-06 11:59:52.502 TRACE 10986 --- [pool-1-thread-1] c.k.m.myapp.tasks.ProcessMonitorTask    : oshi pid=18921, threads=119, getKernelTime()=369277, getUserTime()=886789588
2018-09-06 11:59:57.521 TRACE 10986 --- [pool-1-thread-1] c.k.m.myapp.tasks.ProcessMonitorTask    : oshi pid=18921, threads=119, getKernelTime()=369277, getUserTime()=886795222
```

With oshi-3.8.1:
```
2018-09-06 12:00:47.395 TRACE 14135 --- [pool-1-thread-1] c.k.m.myapp.tasks.ProcessMonitorTask    : oshi pid=18921, threads=0, getKernelTime()=0, getUserTime()=210066
2018-09-06 12:00:52.416 TRACE 14135 --- [pool-1-thread-1] c.k.m.myapp.tasks.ProcessMonitorTask    : oshi pid=18921, threads=0, getKernelTime()=0, getUserTime()=210066
2018-09-06 12:00:57.430 TRACE 14135 --- [pool-1-thread-1] c.k.m.myapp.tasks.ProcessMonitorTask    : oshi pid=18921, threads=0, getKernelTime()=0, getUserTime()=210066
```

Is there something obvious that I am missing here?

**Comment by dbwiddis** [comment]
Which operating system?

**Comment by dbwiddis** [comment]
Based on what I can see from the symptoms, I'm guessing you're on Linux.  I changed the parsing of the `/proc/[pid]/stat` output, and this may be the source of the problem.  If so, it would impact PPID(4), USER_TIME(14), KERNEL_TIME(15), PRIORITY(18), THREAD_COUNT(20), START_TIME(22), VSZ(23), RSS(24).  I suspect an off-by-one error somewhere.  To help me out can you compare OSHI's output to `/proc/18921/stat` (or whichever pid you're testing)?

**Comment by LGH** [comment]
@dbwiddis you are correct, I am using Linux. I should know better than filing incomplete reports, sorry for that. OS is Redhat RHEL 6.7 (Santiago).

```
user@host$ uname -a
Linux hostname 2.6.32-573.22.1.el6.x86_64 #1 SMP Thu Mar 17 03:23:39 EDT 2016 x86_64 x86_64 x86_64 GNU/Linux
user@host$ java -version
java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)
user@host$ cat /proc/18921/stat
18921 (java) S 18920 18906 18906 0 -1 4202496 1160047 0 0 0 82398150 34273 0 0 20 0 119 0 2470487101 11191812096 391111 18446744073709551615 1 1 0 0 0 0 0 0 16800975 18446744073709551615 0 0 17 16 0 0 0 0 0
```

**Comment by dbwiddis** [comment]
I was just about to post, "not only are you on Linux, I bet you're on a kernel older than 3.5".  I see from your output that you are indeed on the 2.6 kernel, so my assumption that `/proc/pid/stat` values contain 52 elements is wrong.  I'll get this fixed.

**Comment by dbwiddis** [comment]
Should be fixed in #611 which is in SNAPSHOT, can you test please?

**Comment by LGH** [comment]
I built and installed 3.9.0-SNAPSHOT and it indeed fixes the issue for me. Thanks a lot for the quick attention @dbwiddis.
